library/src/main/java/pl/rspective/voucherify/android/view/VoucherCheckoutView.java

```diff
@@ -40,6 +40,7 @@
     private Drawable voucherIcon;
     private Drawable validVoucherIcon;
     private Drawable invalidVoucherIcon;
+    private String invalidVoucherErrorMessage;
```

Comment:
@LukaszWiktor Maybe will be better to call this field and setters/getters with more general error message like voucherErrorMessage, because for instance someone could use it also for providing error where edittext is empty where invalidVoucherErrorMessage name is a little bit confusing (at least for me))